.github/workflows/xcodebuild.yml

```diff
@@ -72,9 +72,6 @@ jobs:
         shell: bash
       - name: Codecov
         if: contains(matrix.xcode.flags_for_test, '-enableCodeCoverage YES')
-        run: |
-          if [[ -n "${CODECOV_TOKEN}" ]]; then
```

Comment:
I think that this checking is needed even if `CODECOV_TOKEN` is added, since the documentation said:
https://help.github.com/en/github/automating-your-workflow-with-github-actions/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables
> With the exception of `GITHUB_TOKEN`, secrets are not passed to the runner when a workflow is triggered from a forked repository.

Reply:
Re-added